protocol.md

```diff
@@ -53,13 +53,12 @@ KEYID           | string | No       | No
 
 Functions:
 
-*   PAE() is the
-    [PASETO Pre-Authentication Encoding](https://github.com/paragonie/paseto/blob/master/docs/01-Protocol-Versions/Common.md#authentication-padding),
-    where parameters `type` and `body` are byte sequences:
+*   PAE() is the "Pre-Authentication Encoding", where parameters `type` and
+    `body` are byte sequences:
 
-    ```none
-    PAE(type, body) := le64(2) || le64(len(type)) || type || le64(len(body)) || body
-    le64(n) := 64-bit little-endian encoding of `n`, where 0 <= n < 2^63
+    ```python
+    PAE(type, body) := "DSSEv1 <len(type)> <type> <len(body)> <body>"
+    len(s) := ASCII decimal encoding of the byte length of s, with no leading zeros
```

Comment:
Can you add that <type> and <body> have no leading or trailing space (just like the lengths have no leading zeros)

Reply:
Just pushed an update. Does that help clarify (using ` + SP + ` to make it more clear that a single space is used)?